tahoe.mk

```diff
@@ -24,131 +8,42 @@ tahoe.exec.edxapp:   ## Execute a command in both LMS and Studio (edxapp contain
 	make COMMAND='$(COMMAND)' SERVICE=lms tahoe.exec.single
 	make COMMAND='$(COMMAND)' SERVICE=studio tahoe.exec.single
 
-tahoe.install-pip.edxapp:  ## Install a pip package in both of LMS and Studio
-	make COMMAND='pip install $(PACKAGE)' SERVICE=lms tahoe.exec.single
-	make COMMAND='pip install $(PACKAGE)' SERVICE=studio tahoe.exec.single
-
-tahoe.chown.dir:  # Fix permissions for a single directory
-	find $(DIRECTORY) \! -user $(USER) -print -exec sudo chown -R $(USER) {} \;
-
-tahoe.chown:  ## Fix an annoying docker permission issue in both `edx-platform` and `src`
-	make DIRECTORY='$(DEVSTACK_WORKSPACE)/edx-platform/' tahoe.chown.dir
-	make DIRECTORY='$(DEVSTACK_WORKSPACE)/src/' tahoe.chown.dir
-
-tahoe.theme.compile:  ## Compile the static assets of the theme
-	make COMMAND='make requirements' tahoe.exec.edxapp
-	make lms-static
-	make studio-static
-
-tahoe.theme.reset:  ## Removes and re-clone the theme with Tahoe branches
-	make tahoe.chown
-	rm -rf $(THEMES_DIR)
-
-	# Remove and recreated the customer themes css directory
-	sudo rm -rf $(DEVSTACK_WORKSPACE)/src/themes/customer_themes
-	mkdir -p $(DEVSTACK_WORKSPACE)/src/themes/customer_themes/  # Make it writeble for the user
-
-	git clone git@github.com:appsembler/edx-theme-codebase.git $(THEMES_DIR)/edx-theme-codebase
-	cd $(THEMES_DIR)/edx-theme-codebase && git checkout hawthorn/master
-
-	git clone git@github.com:appsembler/edx-theme-customers.git $(CUSTOMER_THEME_DIR)
-	cd $(CUSTOMER_THEME_DIR) && git checkout hawthorn/tahoe
+tahoe.chown:  ## Fix annoying docker permission issues
+	sudo chown $(USER) -R $(DEVSTACK_WORKSPACE)
```

Comment:
```suggestion
	sudo chown -R $(USER) $(DEVSTACK_WORKSPACE)
```

Reply:
Why @melvinsoft? Doesn't it work on Mac?